src/usr/local/www/system_advanced_network.php

```diff
@@ -106,8 +119,8 @@
 		if ($_POST['disablelargereceiveoffloading'] == "yes") {
 			$config['system']['disablelargereceiveoffloading'] = true;
 		} else {
-			unset($config['system']['disablelargereceiveoffloading']);
-		}
+			unset($config['system']['disablelargereceiveoffloading']); hi		
```

Comment:
It was introduced by the second commit on this PR, aff01dbd97c10731b98bc70c32ac19822c95d304